### DISCUSSION TITLE
Is there a way to import prompts?

### DISCUSSION THREAD

**Discussion by KewkLW** [rewbs/sd-parseq#119] [General]
Trying to find a way to just give parseq a list of prompts to add rather than click add over and over and copy paste over and over.

**Comment by rewbs** [comment]
Hi, currently the only way to bulk-import prompts would be to edit the Parseq JSON directly and re-import it. 

What format would you like to see supported for prompt imports? Deforum's JSON format? A flat list?

**Comment by KewkLW** [comment]
Thanks for the reply! 

A flat list is the most user friendly way of handling it IMO.
